src/pages/VaccineModulePage.js

```diff
@@ -0,0 +1,264 @@
+    const fontStyleLarge = { ...greyTextStyleLarge, color: SUSSOL_ORANGE };
+    const fontStyleSmall = { ...greyTextStyleSmall, color: SUSSOL_ORANGE };
+
+    if (!hasStock) return <Text style={fontStyleSmall}>Empty Fridge</Text>;
+    return (
+      <TouchableOpacity style={fridgeInfoSectionStyle} onPress={onPress}>
+        <Text style={fontStyleSmall}>Total Stock:</Text>
+        <Text style={fontStyleLarge}>{fridgeStock}</Text>
+        {this.renderIcon('angle-double-right', CHERVON_ICON_STYLE)}
+      </TouchableOpacity>
+    );
+  };
+
+  /* Fridge and all of it's components */
+  renderFridge = fridge => {
+    const { sectionStyle, fridgeInfoSectionStyle } = localStyles;
+    const { selectedFridgeCode } = this.state;
+    const isFridgeSelected = fridge.code === selectedFridgeCode;
+
+    return (
+      <View key={fridge.code}>
+        <View style={[sectionStyle, { flexDirection: 'column', alignItems: 'stretch' }]}>
+          <View style={fridgeInfoSectionStyle}>
+            {this.renderFridgeName(fridge, isFridgeSelected)}
+            {this.renderTemperature(30, fridge.getCurrentTemperature())}
+
+            {fridge.isInBreach ? this.renderIcon('warning', BREACH_ICON_STYLE) : null}
+            <View style={[fridgeInfoSectionStyle, { justifyContent: 'flex-end', flexGrow: 1 }]}>
+              {this.renderFridgeExtraInfo(fridge)}
+              {this.renderFridgeStock(fridge)}
+            </View>
+          </View>
+          {isFridgeSelected && (
+            <View style={{ height: 250, alignSelf: 'stretch' }}>
+              <VaccineChart {...fridge.getTemperaturePoints()} />
+            </View>
+          )}
+        </View>
+      </View>
+    );
+  };
+
+  render() {
+    const { fridges } = this.state;
+    const { navigateTo } = this.props;
+
+    const menuButtons = [
+      { text: 'Customer Invoice', onPress: () => console.log('Customer Invoice') },
+      { text: 'Supplier Invoice', onPress: () => console.log('Supplier Invoice') },
+      { text: 'Order Stock', onPress: () => console.log('Order Stock') },
+      {
+        text: 'Manage Stock',
+        onPress: () => navigateTo('manageVaccineItems', 'Manage Vaccine Items'),
+      },
+    ];
+
+    const hasFridges = fridges.length > 0;
+    const {
+      pageContainerStyle,
+      sectionStyle,
+      imageStyle,
+      menuButtonStyle,
+      greyTextStyleLarge,
+      menuButtonTextStyle,
+    } = localStyles;
+
+    return (
+      <View style={pageContainerStyle}>
+        <View style={sectionStyle}>
+          <Image
+            style={imageStyle}
+            resizeMode="contain"
+            source={require('../images/menu_vaccines.png')}
+          />
+          {menuButtons.map(props => (
+            <Button style={menuButtonStyle} textStyle={menuButtonTextStyle} {...props} />
+          ))}
+        </View>
+        {hasFridges && fridges.map(this.renderFridge)}
+        {!hasFridges && <Text style={[greyTextStyleLarge]}>NO CONFIGURED FRIDGES</Text>}
```

Comment:
Should be centered in this area, and maybe lower opacity?

Reply:
referenced at the bottom of pr